### DISCUSSION TITLE
General discussion

### DISCUSSION THREAD

**Discussion by oscbyspro** [oscbyspro/DiffableTextViews#8] [General]
## Welcome 👋

This is a thread for questions, suggestions, and comments.

**Comment by sageibra** [comment]
looks great but ios 15 is not cool dude

**Reply by oscbyspro** [reply]
Thanks, but there is not much I can do about it besides wrapping everything in @available and confusing people. Reason? iOS 15 is when Apple released the FormatStyle protocol: https://developer.apple.com/documentation/foundation/formatstyle. The numeric text style uses this protocol because it is more accurate for large numbers, up to 38 digits, which I want for my use case. You might think it is possible to use NumberFormatter and achieve the same result, but it is not: https://stackoverflow.com/questions/43538661/generatesdecimalnumbers-for-numberformatter-does-not-work.
